configuration/configuration.go

```diff
@@ -255,6 +253,73 @@ func GetKeycloakTestUserSecret() string {
 	return viper.GetString(varKeycloakTesUserSecret)
 }
 
+// GetKeycloakEndpointAuth returns the keycloak auth endpoint set via config file or environment variable.
+// If nothing set then in Dev environment the defualt endopoint will be returned.
+// In producion the endpoint will be calculated from the request by replacing the last domain/host name in the full host name.
+// Example: api.service.domain.org -> sso.service.domain.org
+func GetKeycloakEndpointAuth(req *goa.RequestData) (string, error) {
+	return getKeycloakEndpoing(req, varKeycloakEndpointAuth, devModeKeycloakEndpointAuth, "auth")
+}
+
+// GetKeycloakEndpointToken returns the keycloak token endpoint set via config file or environment variable.
+// If nothing set then in Dev environment the defualt endopoint will be returned.
+// In producion the endpoint will be calculated from the request by replacing the last domain/host name in the full host name.
+// Example: api.service.domain.org -> sso.service.domain.org
+func GetKeycloakEndpointToken(req *goa.RequestData) (string, error) {
+	return getKeycloakEndpoing(req, varKeycloakEndpointToken, devModeKeycloakEndpointToken, "token")
+}
+
+// GetKeycloakEndpointUserInfo returns the keycloak userinfo endpoint set via config file or environment variable.
+// If nothing set then in Dev environment the defualt endopoint will be returned.
+// In producion the endpoint will be calculated from the request by replacing the last domain/host name in the full host name.
+// Example: api.service.domain.org -> sso.service.domain.org
+func GetKeycloakEndpointUserInfo(req *goa.RequestData) (string, error) {
+	return getKeycloakEndpoing(req, varKeycloakEndpointUserinfo, devModeKeycloakEndpointUserinfo, "userinfo")
+}
+
+func getKeycloakEndpoing(req *goa.RequestData, endpointVarName string, devModeEndpoint string, pathSufix string) (string, error) {
+	if viper.IsSet(endpointVarName) {
+		return viper.GetString(endpointVarName), nil
+	}
+	if IsPostgresDeveloperModeEnabled() {
```

Comment:
Why would you have the evn var set up locally if it's not for development purposes? :)
I think it's more useful to be able to override everything by dev env if you need to tests another Keycloak in your dev env.